template/partial/impact_of_assault/impact_1.html

```diff
@@ -0,0 +1,12 @@
+<table class="ui large very padded definition table">
+    <tbody>
+      <tr>
+        <td class="three wide dark-orange" style="height: 64px;"></td>
+      </tr>
+      <tr>
+		<td class="dark-green">
+			<p>Sexual assault can have many effects on the victim – and it affects each person differently. It is important to recognize that all of the reactions are NORMAL. They are NORMAL reactions to an ABNORMAL situation. They are symptoms of a traumatic experience that your body and mind are trying to deal with - like running a fever to fight off infection. While the symptoms are normal, it’s okay to seek help</p>
```

Comment:
Again a github issue.It looks fine on any other editor. I have rechecked it thrice.
<img width="1280" alt="screen shot 2017-06-16 at 13 41 54" src="https://user-images.githubusercontent.com/13470643/27217945-a3fd2dde-5299-11e7-9f8a-a2001cf14978.png">